tests/lean/parserRecovery.lean

```diff
@@ -0,0 +1,88 @@
+import Lean
+
```

Comment:
Should have a module docstring; https://github.com/leanprover/lean4/blob/master/doc/dev/testing.md#writing-good-tests

Reply:
You're right!